packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -194,46 +191,46 @@ class InspectorController extends DisposableController
   final bool isSummaryTree;
 
   /// Parent InspectorController if this is a details subtree.
-  InspectorController parent;
+  InspectorController? parent;
```

Comment:
Nullable, because assigned externally.

Reply:
also not all controllers have a parent controller. only the details one does.